src/eventbridge/send.ts

```diff
@@ -3,24 +3,24 @@ import { EventEntry } from './EventEntry';
 import { Entries } from './Entries';
 import { SendResponse } from './SendResponse';
 import logger from '../observability/logger';
+import { Differences } from '../utils/differences';
 import { TestActivity } from '../utils/testActivity';
 
 const eventbridge = new EventBridge();
-const sendEvents = async (testResults: TestActivity[]): Promise<SendResponse> => {
+const sendEvents = async (events: Array<Differences | TestActivity>, type: string): Promise<SendResponse> => {
   logger.info('sendEvents starting');
-  logger.info(`${testResults.length} ${testResults.length === 1 ? 'event' : 'events'} ready to send to eventbridge.`);
+  logger.info(`${events.length} ${events.length === 1 ? 'event' : 'events'} ready to send to eventbridge.`);
 
   const sendResponse: SendResponse = {
     SuccessCount: 0,
     FailCount: 0,
   };
 
-  for (let i = 0; i < testResults.length; i++) {
-    const result = testResults[i];
+  for (let i = 0; i < events.length; i++) {
+    const event = events[i];
     const entry: EventEntry = {
       Source: process.env.AWS_EVENT_BUS_SOURCE,
-      // eslint-disable-next-line security/detect-object-injection
-      Detail: `{ "testResult": "${JSON.stringify(result)?.replace(/"/g, '\\"')}" }`,
+      Detail: `{ "testResult": "${JSON.stringify(event)?.replace(/"/g, '\\"')}" }, type: ${type}`,
```

Comment:
Will the inserts be sending the `type: ${type}` or is it only needed for your modify changes?

Reply:
we need to send it for insert as well, it should have three values:
- `completion` for VTA tests
- `contingency` for contingency tests
- `amendment` for amendments
Did want to put them in an enum though, so yeah I'll change that

Reply:
That makes sense, an enum would be good